submit/submit.go

```diff
@@ -23,9 +23,10 @@ import (
 )
 
 type cliargs struct {
-	Image     string   `arg:"-i,required,help:ECR image like $acct.dkr.ecr.$region.amazonaws.com/$tag"`
+	Image     string   `arg:"-i,required,help:image like $acct.dkr.ecr.$region.amazonaws.com/$image:$tag or $image:$tag"`
+	Registry  string   `arg:"env" help:"Docker image registry, defaults to $acct.dkr.ecr.$region.amazonaws.com"`
 	Role      string   `arg:"-r,required,help:existing role name"`
-	Region    string   `arg:"help:region for batch setup"`
+	Region    string   `arg:"env" help:"region for batch setup"`
```

Comment:
I think this needs to be, e.g.
```
`arg:"env,help:AWS region for batch setup"`
```
otherwise, the help wont show.

Reply:
and what do you think about making this `env:AWS_DEFAULT_REGION` ?

Reply:
Good idea, just pushed that change. Thanks.